Merge Conflict/Assets/Scripts/AudioManager/AudioManager.cs

```diff
@@ -0,0 +1,218 @@
+           get component when throwing one away (upgrade), buy component, selling wrong component,  ... )  */
+
+    // TODO: don't play dropComponent Sound when selling a component or throwing it in the trash can
+
+    private Coroutine playNextSongCoroutine;
+    private Coroutine footstepCoroutine;
+
+    private void Awake()
+    {
+        if (_instance)
+        {
+            Destroy(gameObject);
+        }
+
+        _instance = this;
+
+        DontDestroyOnLoad(gameObject);
+    }
+
+    private void Start()
+    {
+        playNextSongCoroutine = StartCoroutine(PlayNextSong());
+    }
+
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.M))
+        {
+            if (playNextSongCoroutine != null)
+            {
+                StopCoroutine(playNextSongCoroutine);
+            }
+            playNextSongCoroutine = StartCoroutine(PlayNextSong());
+        }
+    }
+
+    private IEnumerator PlayNextSong()
+    {
+        while (true)
+        {
+            var clip = GetNextSong();
+
+            if (clip)
+            {
+                backgroundAudioSource.clip = clip;
+                backgroundAudioSource.Play();
+
+                yield return new WaitForSecondsRealtime(backgroundAudioSource.clip.length);
+            }
+
+            yield return null;
+        }
+    }
+
+    private AudioClip GetNextSong()
+    {
+        switch (playMode)
+        {
+            case PlayMode.Random:
+                return playlist.NextRandomSong();
+
+            case PlayMode.Sequential:
+                return playlist.NextSong();
+
+            default:
+                return null;
+        }
+    }
+
+    public void PlayButtonClickSound()
+    {
+        buttonClickAudioSource.PlayOneShot(buttonClickSound);
+    }
+
+    public void PlayOpenMenuSound()
+    {
+        openMenuAudioSource.PlayOneShot(openMenuSound);
+    }
+
+    public void PlayCloseMenuSound()
+    {
+        closeMenuAudioSource.PlayOneShot(closeMenuSound);
+    }
+
+    public void PlayMergeSound()
+    {
+        mergeAudioSource.PlayOneShot(mergeSound[0]);
+    }
+
+    public void PlayCombineComponentsSound()
+    {
+        combineComponentsAudioSource.PlayOneShot(combineComponentsSound);
+    }
+
+    public void PlaySellSound()
+    {
+        sellAudioSource.PlayOneShot(sellSound);
+    }
+
+    public void PlayTrySellWrongComponentSound()
+    {
+        sellAudioSource.PlayOneShot(trySellWrongComponentSound);
+    }
+
+    public void PlayThrowAwaySound()
+    {
+        throwAwayAudioSource.PlayOneShot(throwAwaySound);
+    }
+
+    public void PlayDropComponentSound()
+    {
+        dropComponentAudioSource.PlayOneShot(dropComponentSound);
+    }
+
+    public void PlayPickUpComponentSound()
+    {
+        pickUpComponentAudioSource.PlayOneShot(pickUpComponentSound);
+    }
+
+    public void StartFootstepLoop()
```

Comment:
make it more clear that the component footstep is meant

Reply:
done